lib/elixir/lib/option_parser.ex

```diff
@@ -467,9 +467,9 @@ defmodule OptionParser do
   (if the value is `true` or `false`, respectively),
   and all other values are converted using `Kernel.to_string/1`.
 
-  It is advised to pass to `to_argv/2` the same set of `options`
+  It is advised to pass to `to_argv/2` the same set of `:options`
```

Comment:
I think this `options` refers to the argument, so it should be `options` and not `:options`

Reply:
oops. fixed